.vscode/tasks.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
**nitpick:** You should not keep VSC related files in repository, as not every developer that you will work will have the same IDE. Configuration should not be IDE specific.